### ISSUE TITLE
shake down dependency tree

### ISSUE THREAD

**Issue by Inlife** [Inlife/nexrender#17]
Currently, nexrender has a very deep and big dependency tree.
Installation of nexrender takes a while, especially on HDD.

Most of the "leaves" of the dep. tree caused by `request`, `download`, and `express.js` npm modules.

I have a plan to either search for some more lightweight alternatives or to rewrite those parts natively (this option opens up a whole new world for bugs). 

Guys, if you have any thoughts/suggestions, just post 'em there. :)

**Comment by southernsun** [comment]
low priority if you ask me. disk space is not really an issue nowadays and the hit is only once (when installing npm packages).

**Comment by Inlife** [comment]
@southernsun, this is kinda long-term feature, so you are right.
